test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        //empty list
+        assertFalse( false);
+
+        //only one object
+        assertTrue(null, true);
+        assertFalse("1", false);
+        assertFalse("",false);
+        assertFalse("abc",false);
```

Comment:
These 4 lines have the same problem as the above case. You did not test the `isAnyNull` method. Please ensure you understand how the assert methods in JUnit works.